opengever/apiclient/client.py

```diff
@@ -72,3 +74,16 @@ def listing(self, name, columns=[], raw=False, **kwargs):
 
     def update_object(self, **data):
         return self.session().patch(self.url, json=data).ok
+
+    @autowrap
+    def create_document(self, title, io, content_type, filename):
+        return self.session().post(self.url, json={
```

Comment:
I'd appreciate a docstring explaining the parameter, especially the type of the parameter `io`.

Reply:
Thanks!